prow/proxy-common.inc

```diff
@@ -57,7 +57,7 @@ if [[ -n "${GOOGLE_APPLICATION_CREDENTIALS:-}" ]]; then
 
   # Use RBE when logged in.
   BAZEL_BUILD_RBE_INSTANCE="${BAZEL_BUILD_RBE_INSTANCE-projects/istio-testing/instances/default_instance}"
-  BAZEL_BUILD_RBE_CACHE="${BAZEL_BUILD_RBE_CACHE:-grpcs://remotebuildexecution.googleapis.com}"
+  BAZEL_BUILD_RBE_CACHE="${BAZEL_BUILD_RBE_CACHE-grpcs://remotebuildexecution.googleapis.com}"
```

Comment:
Use the default rbe setting when BAZEL_BUILD_RBE_CACHE=="" ? NBD but I am curious why you make this change

Reply:
This is change to match style used in the line above (see: https://github.com/istio/proxy/pull/2995), so that it doesn't override an empty value if someone wants to disable this feature.

Reply:
So the current behavior is that disable cache only if `BAZEL_BUILD_RBE_CACHE` is unset. I don't have a preference as long as all existing runs are good